src/geo/transform.js

```diff
@@ -136,6 +144,7 @@ class Transform {
         this._projMatrixCache = {};
         this._alignedProjMatrixCache = {};
         this._fogTileMatrixCache = {};
+        this._distanceMatrixCache = {};
```

Comment:
nit: For naming consistency, maybe rename one or the other to have the same naming structure, either `_distanceMatrixCache` -> `_distanceTileMatrixCache`, or `_fogTileMatrixCache` -> `_fogMatrixCache` (I find that keeping the name of the units in which it transforms it to be a useful hint, so maybe the first option?).